minifi-commons/minifi-commons-schema/src/main/java/org/apache/nifi/minifi/commons/schema/ConfigSchema.java

```diff
@@ -186,18 +186,38 @@ public FlowControllerSchema getFlowControllerProperties() {
         return flowControllerProperties;
     }
 
+    public void setFlowControllerProperties(final FlowControllerSchema flowControllerProperties)
+    {
+        this.flowControllerProperties = flowControllerProperties;
+    }
+
```

Comment:
I can take this part of this PR if we'd prefer. I don't think it hurts anything because a lot of fields have setters, just not all of them. But I can pull this out and submit with a follow up PR with more of MINIFI-536 if preferred.

Reply:
I'm fine with keeping them in this PR, but the leading `{` brace has to be on the previous line or it causes a CheckStyle violation

Reply:
Good catch. Updated to fix checkstyle